vnext/proposals/active/mouse-reconcile-desktop.md

```diff
@@ -0,0 +1,91 @@
+          if (pressed) {
+            backgroundColor = 'lightblue';
+          }
+          return {
+            padding: 10,
+            margin: 10,
+            borderWidth: 1,
+            borderColor: focused ? 'red' : null,
+            backgroundColor,
+            outlineWidth: 0
+          };
+        }}
+      >
+      <Text>Nested pressables</Text>
+    </Pressable>
+  </Pressable>
+</View>
+```
+
+## Outstanding issues
+There are some outstanding issues that need to be address specifically for desktop scenarios when it comes to fully implementing mouse hover and leave behavior. They are as follows:
+
+- Prevent mouse input from being 'eaten' in ScrollView scenarios [issue #5867](https://github.com/microsoft/react-native-windows/issues/5867)
+- Mouse leave event not firing correctly in Scrollable FlatList scenarios [issue #4614](https://github.com/microsoft/react-native-windows/issues/4614)
+- The hover events/indication for overlapping content prioritizes items "closer" in z-order [issue #5901](https://github.com/microsoft/react-native-windows/issues/5901)
+- Remove the tapped event handler on the `View` component [issue #2349](https://github.com/microsoft/react-native-windows/issues/2349)
+
+## onPointerXX callbacks
+The following callbacks are available on `View` component (and get passed through to Pressable) in Windows and macOS to cover the most common use cases where mouse handling would occur.
+
+The below will need to be implemented for macOS as well [issue TBD]()
+
+| API | Args | Returns | Description |
+|:---:|:----:|:-------:|----|
+| onPointerOver | IPointerEvent | void | Fires when a pointing device is moved within the hit test boundaries of a element. |
+| onPointerLeave | IPointerEvent | void | Fires when a pointing device is moved out of the hit test boundaries of an element and all of its children. |
+
+### Example
+
+``` js
+<View onPointerOver={this._onPointerOver} />
+
+private onPointerOver_ = (event: IPointerEvent) => {
+  this.setState({ lastKeyDown: event.nativeEvent.key });
+};
+```
+
+### Declarative properties
+
+To co-ordinate the handoffs of these pointer events between the native layer and the JS layer, we are also introducing 2 corresponding properties on the `View`(?) component. Those are:
+
+| Property | Type | Description |
+|:---:|:----:|----|
+| pointerOverEvents | IHandledPointerEvents[] | Specifies the pointer over events that are handled in the JS layer by the onPointerOver/onPointerOverCapture events |
+| pointerLeaveEvents | IHandledPointerEvents[] | Specifies the button or buttons that are handled in the JS layer by the onPointerOut/onPointerOutCapture events |
+
+Where `IHandledPointerEvents` is a new type which takes the following parameters:
+- A constrained string parameter named `pointerState` to declare the pointer state (Over, Hover, Out) that is of interest to the JS layer
+- An `eventPhase` parameter of type `EventPhase` to declare the routing phase of interest to the JS layer.
```

Comment:
Is this `EventPhase` just the generic one that is also being used in KeyboardEvents? Meaning, the parameters indicate bubbling/tunneling like the web, correct? Maybe good to add links to the React docs for these to make it simpler for implementation coherence.